ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -218,7 +218,6 @@ StatusDialog {
                             root.swapInputParamsForm.fromTokenAmount = amount
                         }
                     }
-                    onValueValidChanged: d.fetchSuggestedRoutes()
```

Comment:
because earlier we checked if payPanel.valueValid and only then fetched routes but now we depend on isForCorrectlyFilled instead and hence refetching on onValueValidChanged is not needed anymore. 

i had to remove the check for payPanel.valueValid cause it was only valid if `amount entered < balance`, but we want to fetch routes even if `amount entered > balance`